src/utils/event-utils.ts

```diff
@@ -31,6 +31,25 @@ export const CAMPAIGN_PARAMS = [
     'rdt_cid', // reddit
 ]
 
+export const EVENT_TO_PERSON_PROPERTIES = [
```

Comment:
I think this could do with a comment or a better name... 

I think it's saying "these are event properties that we will copy to person properties"
but I don't know for sure 🙈

Reply:
I don't disagree but it's copied directly from plugin server with that name, I'll add a comment though!